src/allotropy/parsers/utils/values.py

```diff
@@ -20,6 +20,15 @@ def try_float(value: Optional[str]) -> Optional[float]:
         return None
 
 
+def try_str(value: Any) -> Optional[str]:
+    if not value:
+        return None
+    try:
+        return str(value or "")
+    except ValueError:
+        return None
```

Comment:
How could `str` of any non-`None` value throw `ValueError`? Also, since `value` is guaranteed to be truthy here, `or ""` will never get hit.
```suggestion
    return str(value)
```

Also, could you please write tests for this utility method? Integration tests are nice but shouldn't contribute to coverage.